repository.json

```diff
@@ -128,6 +128,21 @@
 				}
 			]
 		},
+		{
+			"details": "https://github.com/TheSecEng/LSP-copilot",
+			"name": "LSP-copilot",
+			"labels": [
+				"auto-complete",
+				"code-intelligence",
+				"intellisense"
+			],
+			"releases": [
+				{
+					"sublime_text": ">=4126",
+					"tags": "4126-"
```

Comment:
with this, we should `git tag` a commit as like `4126-0.1.0` for version `0.1.0`

Reply:
I don't see why would you need to use versioned tag in the first place if you don't support ST3.

Reply:
To make it less odd if it gets bumped in the future?

Reply:
The least odd way to do it is to keep the main and latest supported version on the non-versioned (build-less) tags and, once it's needed, add versioned tags for older builds that need to branch out.

Reply:
Not convincing since that's not how LSP is doing.

Reply:
I don't feel that just because LSP is doing that it's the best way to do it.

Some of the LSP-* packages do it the way I've suggested and there are no issues with that. It's more obvious and natural to have normal tags for the current/ongoing branch.

Reply:
Sure I will re-release